src/lazy/expanded/macro_table.rs

```diff
@@ -204,504 +202,195 @@ impl MacroTable {
     pub const FIRST_USER_MACRO_ID: usize = Self::NUM_SYSTEM_MACROS;
 
     fn compile_system_macros() -> Vec<Arc<Macro>> {
```

Comment:
🪧 This method is where the system macro table is bootstrapped, making it the primary benefactor of our refactoring.